oneadapter/src/main/java/com/idanatz/oneadapter/internal/InternalAdapter.kt

```diff
@@ -352,6 +352,10 @@ internal class InternalAdapter(val recyclerView: RecyclerView) : RecyclerView.Ad
     }
     //endregion
 
+    fun getItemViewTypeFromClass(clazz: Class<*>): Int? {
```

Comment:
change to Class < Diffable > for the reason written in 
ViewHolderCreatorsStore.kt

Reply:
and place it below 
fun getItemViewType